.github/workflows/prerelease_linux.yml

```diff
@@ -17,6 +17,8 @@ env:
   TAG: ${{ github.event.release.tag_name }}
   DOCKER_HUB_ID: ${{ secrets.OHAI_DOCKER_HUB_ID }}
   DOCKER_HUB_PASSWORD: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+  DOCKER_PUBLISH: true
+  IS_RELEASE: true
```

Comment:
Now it is inconsistent with OHIs pipelines. I would go with not the best but consistent solution.

Reply:
So remove `IS_RELEASE` or `DOCKER_PUBLISH` or both?

Reply:
Removed `IS_RELEASE` and reverted it back to `PRERELEASE` (and original logic).